<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteEmailDialog</class>
 <widget class="QDialog" name="WriteEmailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="head">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background-color: rgb(244, 244, 244);}
#send{border:none;background-color: rgb(244, 244, 244);}
#accessory{border:none;background-color: rgb(244, 244, 244);}
#send:hover{background-color: rgb(219,219,219);}
#accessory:hover{background-color: rgb(219,219,219);}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>15</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/发送.png</normaloff>:/souce/souce/发送.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="accessory">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>15</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>附件</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/附件.png</normaloff>:/souce/souce/附件.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>720</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="subject">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>主 题:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subjectEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="receiver">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>收件人:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="receiverEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="emailContent">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>720</width>
     <height>340</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>720</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 244, 244);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>样式编辑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>样式编辑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>样式编辑</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="myPic.qrc"/>
  <include location="souce.qrc"/>
 </resources>
 <connections/>
</ui>
